### PR TITLE
Show amounts with two decimals.

### PR DESCRIPTION
**Pull request by Sharqiewicz**
- [x] Show two decimals for Fee collapse
- [x] Fix Fee collapse animation (by only using DaisyUI styles instead of combining several features: useState + details/summary + daisyUI) 

Screenshot 2024-10-10 at 15 58 02

### CODE REVIEW

**Review by ebma — APPROVED** [review]
Nice, the collapse is super smooth now 🙏 
And the rest decimal rounding looks good to me too

File: src/components/FeeCollapse/index.tsx [hunk 1]
```diff
@@ -21,35 +20,29 @@ interface CollapseProps {
...
   };
 
-  const chevron = isOpen ? (
-    <ChevronUpIcon className="w-8 text-blue-700" />
-  ) : (
-    <ChevronDownIcon className="w-8 text-blue-700" />
-  );
-
-  const toAmountFixed = roundDownToSignificantDecimals(new Big(toAmount || 0), 2).toString();
+  const toAmountFixed = roundDownToTwoDecimals(Big(toAmount || 0)).toString();
```
Comment (TorstenStueber) [inline src/components/FeeCollapse/index.tsx:31]: `roundDownToTwoDecimals` already returns a string (and this occurs two more times below).

File: src/components/FeeCollapse/index.tsx [hunk 2]
```diff
@@ -21,35 +20,29 @@ interface CollapseProps {
 }
 
 export const FeeCollapse: FC<CollapseProps> = ({ toAmount, toToken, exchangeRate }) => {
-  const [isOpen, setIsOpen] = useState(false);
   const { trackEvent } = useEventsContext();
+  const isOpenCheckboxRef = useRef<HTMLInputElement>(null);
```
Comment (TorstenStueber) [inline src/components/FeeCollapse/index.tsx:24]: Do we actually need `isOpenCheckboxRef`? It does not seem to be used.

**Comment by Sharqiewicz** [comment]
@TorstenStueber Well spotted! I have improved the code ✅

**Review by TorstenStueber — APPROVED** [review]
👍 🥇

**Pull request merged**